cypress/utils/index.ts

```diff
@@ -54,18 +54,14 @@ export const clickOnPageBtnAndAssertURLandTableResults = (
 /**
  * Assert limit query param and table length after clicking on a page size button
  * @param pageSize new page size
- * @param dataCyPageSizeSelectorToggle data-cy for page size dropdown toggle
- * @param dataCyPageSizeBtn data-cy for page size button to select page size
  * @param dataCyTableRows dat-cy for table rows
  */
 export const clickOnPageSizeBtnAndAssertURLandTableSize = (
```

Comment:
suggestion. You might also be able to use this in `job_logs.ts`. There is a test for changing the page size, but it's not as nice as this util function!